yabause/src/port/qt/ui/UISettings.cpp

```diff
@@ -1,21 +1,21 @@
 /*  Copyright 2005 Guillaume Duhamel
-	Copyright 2005-2006, 2013 Theo Berkau
-	Copyright 2008 Filipe Azevedo <pasnox@gmail.com>
+        Copyright 2005-2006, 2013 Theo Berkau
```

Comment:
Formatting is a mess in kronos, it is lmainly due to the fact I am working in linux, with default formatting rules.
So unless there is a huge rework of the full formatting, if I accept this reformatting in the PR, it will always be a ping pong in future PR. 
As this formatting stuff is reducing visibility on what has to change really, is that possible to split the patches in two patches. First one: open Kronos source and save them with your IDE, it will generate formatting difference that you can add as a separate patch, then goes to your effective changes in a second patch so that review will be easier?